### PR TITLE
PGD re-edit group 11 - rest of reference section

### PR DESCRIPTION
**Pull request by ebgitelman**

### CODE REVIEW

File: product_docs/docs/pgd/5/reference/repsets-membership.mdx [hunk 1]
```diff
@@ -8,10 +8,10 @@ indexdepth: 2
 
 This function adds a table to a replication set.
 
-This adds a table to a replication set and starts replicating changes
-from this moment (or rather transaction commit). Any existing data the table
+This function adds a table to a replication set and starts replicating changes
+at that time (on transaction commit). Any existing data the table
 might have on a node isn't synchronized.
-
+<!-- Check edit >
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/repsets-membership.mdx:14]:
```suggestion

```

File: product_docs/docs/pgd/5/reference/repsets-membership.mdx [hunk 2]
```diff
@@ -8,10 +8,9 @@ indexdepth: 2
 
 This function adds a table to a replication set.
 
-This adds a table to a replication set and starts replicating changes
-from this moment (or rather transaction commit). Any existing data the table
+This function adds a table to a replication set and starts replicating changes
+at that time (on transaction commit). Any existing data the table
```
Comment (ebgitelman) [inline product_docs/docs/pgd/5/reference/repsets-membership.mdx:12]: Is this what was meant? It took reading this a few times to glean what was meant.

Comment (djw-m) [inline product_docs/docs/pgd/5/reference/repsets-membership.mdx:12]:
```suggestion
from the committing of the transaction that contains the call to the function. Any existing data the table
```

**Comment by ebgitelman** [comment]
@djw-m Committed your change, should be ready to go. Thank you!

**Comment by ebgitelman** [comment]
@djw-m Please approve so I can merge.

**Review by djw-m — APPROVED** [review]
LGTM

**Pull request merged**